pkg/kubectl/cmd/util/shortcut_restmapper.go

```diff
@@ -67,7 +69,19 @@ func (e ShortcutExpander) getAll() []schema.GroupResource {
 		}
 	}
 
-	return availableAll
+	// populate all "discovered" resources from the server
+	for availableResource := range availableResources {
```

Comment:
Added a second commit that filters out discovered resources that are not namespaced, however still need to filter out admin-level and potentially unrecoverable resources from the list